### DISCUSSION TITLE
virtnest/intro/index

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2415] [Announcements]
# virtnest/intro/index

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/virtnest/intro/index.html

**Comment by liyuerich** [comment]
这个功能属于标准版，高级版，还是白金版？

**Comment by WANG0608GitHub** [comment]
已添加 Note 在文章末尾：https://docs.daocloud.io/virtnest/intro/index.html#virtnest_1
